### ISSUE TITLE
Compatibility with FastAdapter 4.0.1

### ISSUE THREAD

**Issue by marbat87** [mikepenz/MaterialDrawer#2456]
Hi,

I'm using MaterialDrawer 7.0.0-rc01 and FastAdapter 4.0.1 and they're not compatible since MaterialDrawer calls some methods that don't exist anymore.

For example Drawer.kt calls setNewList with 1 parameter:
```
private fun setItems(drawerItems: List<IDrawerItem<*>>?, switchedItems: Boolean) {
        //if we are currently at a switched list set the new reference
        if (originalDrawerItems != null && !switchedItems) {
            originalDrawerItems = drawerItems
        }
        drawerBuilder.itemAdapter.setNewList(drawerItems ?: ArrayList())
    }
```

But in FastAdapter 4.0.1 the interface method it is defined as (from ItemAdapter.kt):
`fun setNewList(items: List<Model>, retainFilter: Boolean = false): IItemAdapter<Model, Item>`

Thanks in advance

**Comment by mikepenz** [comment]
Thank you very much for this report. We will investigate. It is strange as this version is also used in the FastAdapter sample without issues: https://github.com/mikepenz/FastAdapter/blob/develop/build.gradle#L28

**Comment by AllanWang** [comment]
Is this actually a compilation issue? Kotlin has default variables so that call is fine

**Comment by AllanWang** [comment]
https://github.com/mikepenz/MaterialDrawer/pull/2458

It builds

**Comment by mikepenz** [comment]
I have released v7.0.0-rc02 which uses v4.0.1 of the FastAdapter there should be definitely no more conflict. if you still see issues, please ensure no other dependency might load a different version of the FastAdapter
